AUTHORS.rst

```diff
@@ -41,18 +40,17 @@ Many thanks for your contributions!
 There are also a few modules or functions incorporated from other
 authors and projects:
 
-* ``utils.iso_precision`` includes something like 10 lines of Arrow's, code,
-  which is written by Chris Smith and under Apache license
+* The function ``ics.utils.iso_precision`` includes something like 10 lines of
+  Arrow's, code, which is written by Chris Smith and under Apache license.
 
 
-A big part of the code was written at `UrLab <http://urlab.be>`_, an awesome
-hackerspace in `ULB <http://ulb.ac.be>`_ in Brussels, Belgium.
-A big part of the code was written at `UrLab <http://urlab.be>`_, an awesome hackerspace in `ULB <http://ulb.ac.be>`_ in Brussels, Belgium.
+A big part of the code was written at `UrLab <https://urlab.be>`_, an awesome
+hackerspace at `ULB <https://ulb.be>`_ in Brussels, Belgium.
 
 Thanks to:
 
-* `arrow <http://crsmithdev.com/arrow/>`_ which provides a nice API for dates, times and deltas,
-* `python-dateutil <http://labix.org/python-dateutil>`_ for parsing timezones from VTIMEZONE blocks,
-* `requests <http://python-requests.org/>`_ for giving me inspiration from it's beautiful and pythonic API (and its doc too),
-* `six <http://pythonhosted.org/six/>`_ to have made the python3 transition easier,
+* `arrow <https://arrow.readthedocs.io/en/latest>`_ which provides a nice API for dates, times and deltas,
```

Comment:
Maybe also drop this line, as the API turned out to be not-so-nice for floating datetimes (i.e. without timezone, which is explicitly allowed by the spec). For this reason, `arrow` was completely removed and we're now using plain but correct `datetimes`. See #222.

Reply:
Yes!

Reply:
```suggestion
```